pkg/utils/staking.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package utils
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/staking"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+)
+
+func NewStakingCertAndKeyBytes() ([]byte, []byte, error) {
```

Comment:
no need for this function, just directly use staking.NewCertAndKeyBytes on the caller